tests/e2e/specs/setup-mc/step-1-accounts.test.js

```diff
@@ -398,12 +403,15 @@ test.describe( 'Set up accounts', () => {
 
 		test.describe( 'When all accounts are connected', async () => {
 			test.beforeAll( async () => {
-				await setUpAccountsPage.mockJetpackConnected();
-				await setUpAccountsPage.mockGoogleConnected();
 				await setUpAccountsPage.mockAdsAccountConnected();
-				await setUpAccountsPage.mockAdsStatusClaimed();
 				await setUpAccountsPage.mockMCConnected();
 
+				await setUpAccountsPage.fulfillAdsAccountStatus( {
+					has_access: true,
+					invite_link: '',
+					step: 'link_merchant',
```

Comment:
This is very similar to the response from `mockAdsStatusClaimed()` except for the `step` that is returned. This matches what the API returns from the `ads/account-status` endpoint:

<img width="685" alt="image" src="https://github.com/user-attachments/assets/64e9c99f-e294-4323-ae6a-31855febb9ba">

However, when the the Ads account is in this state, the `ads/connections` response would look something like this:

<img width="904" alt="image" src="https://github.com/user-attachments/assets/e066bb2a-6447-4fbb-a45c-b0df447893c0">

Making these two responses consistent would make for a more robust test, I think.